### PR TITLE
OCPBUGS-12280: clarifying cpumanager-enabled diction

### PR DESCRIPTION
**Pull request by jmanthei**
Version(s):

Current
Issue:

https://issues.redhat.com/browse/OCPBUGS-12280
Link to docs preview:

https://docs.openshift.com/container-platform/4.12/scalability_and_performance/using-cpu-manager.html#topology_manager_policies_using-cpu-manager-and-topology_manager

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/topology-manager-policies.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 Topology Manager aligns `Pod` resources of all Quality of Service (QoS) classes by collecting topology hints from Hint Providers, such as CPU Manager and Device Manager, and using the collected hints to align the `Pod` resources.
 
-Topology Manager supports four allocation policies, which you assign in the `cpumanager-enabled` custom resource (CR):
+Topology Manager supports four allocation policies, which you assign in the KubeletConfig customer resource (CR) named cpumanager-enabled:
```
Comment (bergerhoffer) [inline modules/topology-manager-policies.adoc:11]: Same comments as the other file about the typo and marking up the items properly

**Review by bergerhoffer — COMMENTED** [review]
A few things to fix!

File: modules/setting-up-topology-manager.adoc [hunk 2]
```diff
@@ -7,7 +7,7 @@
 [id="seting_up_topology_manager_{context}"]
 = Setting up Topology Manager
 
-To use Topology Manager, you must configure an allocation policy in the `cpumanager-enabled` custom resource (CR). This file might exist if you have set up CPU Manager. If the file does not exist, you can create the file.
+To use Topology Manager, you must configure an allocation policy in the KubeletConfig customer resource (CR) named cpumanager-enabled. This file might exist if you have set up CPU Manager. If the file does not exist, you can create the file.
```
[the remark this answers is not in the document]
Comment (jmanthei) [reply modules/setting-up-topology-manager.adoc]: Thank you for the reminder not to click the "commit suggestion" button.

**Comment by jmanthei** [comment]
@stevsmit thanks, fixed it with help!! /label merge-review-needed

**Review by skrthomas — APPROVED** [review]
Merge review lgtm

**Pull request merged**